Dockerfile

```diff
@@ -4,6 +4,7 @@ FROM debian:sid
 ## enable some apache mods (the ln -s lines)
 ## cleanup freepascal since it is no longer needed after wattsi build
 RUN apt-get update && \
+    apt-get install -y python python-dev python-pip python-virtualenv && \
```

Comment:
We'll also want to update the README's list of requirements. Although if we do it right we can point out that they're optional if you use the `--no-update` flag.